docs/source/registries.rst

```diff
@@ -0,0 +1,24 @@
+Registries
+==========
+
+All models in Curated Transformers are added to a registry. Each auto class uses
```

Comment:
Do we want to add a comment mentioning what happens when more than one model registers for the same model type (the one that was registered first gets picked)? Or just implicitly leave that as an impl detail?

Reply:
It crossed my mind, but I'm not sure what happens now and what behavior we would prefer. So maybe keep it underspecified for now until we know what we want?

Reply:
I think we just iterate over the backing dictionary in insertion order. But keeping it under specified for now is fine, I think.

Reply:
That still doesn't tell us anything, because the order in which the entry points are inserted in the dictionary is determined by the order in which `importlib.metadata` discovers packages. This is probably in turn determined by `sys.path`. I haven't looked into how this is implemented, but if that's the case the order is undetermined, because the user controls it.